### ISSUE TITLE
Changing the font to an external one (system path) in the editor settings prints errors

### ISSUE THREAD

**Issue by capnm** [godotengine/godot#23123]
**Godot version:**

00125ef8df0c7964a71c6781bd65d0e242187b04

**OS/device including version:**

Linux

**Issue description:**

```
ERROR: getornull: Condition ' !id_map.has(p_rid.get_data()) ' is true. returned: __null
   At: ./core/rid.h:164.
ERROR: getornull: Condition ' !id_map.has(p_rid.get_data()) ' is true. returned: __null
   At: ./core/rid.h:164.
ERROR: getornull: Condition ' !id_map.has(p_rid.get_data()) ' is true. returned: __null
   At: ./core/rid.h:164.
...
```
In some cases, selecting a font (in the file selection dialog) has even closed the editor's settings box.

**Steps to reproduce:**
![image](https://user-images.githubusercontent.com/4047289/47181326-ff471f00-d322-11e8-955d-8858fd7a7e52.png)

**Comment by kakoeimon** [comment]
This became the default editor in my project. All text is turned to this white rectangle when the editor is fully loaded.
Ubuntu 18.04 64bit
Nvidia GT540M 390.77

**Comment by akien-mga** [comment]
I can't reproduce the issue in the current master branch (fb9c837). The only problem I see is that the preview is broken, but no errors are printed.

**Comment by capnm** [comment]
Afaict nothing changed 6ad4f16 :

![image](https://user-images.githubusercontent.com/4047289/51718450-894f0400-2045-11e9-8a61-ae27a454d874.png)

The preview for external folders should be probably disabled.

**Comment by akien-mga** [comment]
Ah I had missed the detail that it was about fonts in system paths and not included in `res://`.

**Comment by somnivore** [comment]
Getting this error spam when setting a font from res:// to a DynamicFont.

![image](https://user-images.githubusercontent.com/28724859/52389715-7c291080-2a5a-11e9-9866-f0aa53cbd9cc.png)

**Comment by volzhs** [comment]
it seems fixed by f436047cf584271edea0e87d8765e1a0c72dcd66
